backend/vectorize_utils.mli

```diff
@@ -81,3 +81,13 @@ module Vectorized_instruction : sig
 
   val make_default : arg_count:int -> res_count:int -> Operation.t -> t
 end
+
+(** Given two registers of non-vector types, return true iff there exist a vector type
+    that can contain both of them. Currently distinguishes between [Val] and other
+    types. Mixing [Val] with non-Val in a vector is not yet supported. *)
+val vectorizable_machtypes : Reg.t -> Reg.t -> bool
+
+val vectorize_machtypes : Reg.t list -> Cmm.machtype_component
+(* CR-someday gyorsh: [vectorizable_machtypes] should take a [pack] instead of a
+   pair, to handle longer vectors, and to present a uniform interface with
```

Comment:
Is taking a list sufficient?

Reply:
I was referring to generalizing `vectorizable_machtypes r1 r2` to `vectorizable_machtypes reg_list`. Why would it not be enough?

Reply:
Sure, I mean the cr-someday can be deleted then

Reply:
The function `vectorize_machtypes` in the preview already uses a pack. The CR is talking about `vectorizable_machtypes` on Line 88, right above the preview, not visible in the preview, and does not take a list as input. This should really be the same function, and maybe should also live in a target-specific file. I prefer to clean this up in a follow up PR, if that's okay with you, @TheNumbat ?

Reply:
slightly improved the CR before resolving this comment.